ui/pages/home/home.component.js

```diff
@@ -338,12 +254,6 @@ export default class Home extends PureComponent {
 
     if (notificationClosing && !prevState.notificationClosing) {
       attemptCloseNotificationPopup();
-    } else if (
```

Comment:
If this logic previously navigated based on certain branches on update, but we're now only triggering that once we reach the render, could that change the behaviour if the below branches fire first?